src/constants.rs

```diff
@@ -66,4 +66,27 @@ pub mod gas_cost {
     pub fn log_dynamic_gas_cost(size: u32, topic_count: u32) -> i64 {
         (super::gas_cost::LOG * topic_count as i64) + (8 * size as i64)
     }
+
+    pub fn exponent_byte_size(exponent: u64) -> u32 {
```

Comment:
This should be private
```suggestion
   fn exponent_byte_size(exponent: u64) -> u32 {
```